PXWeb/Resources/Languages/pxlang.xml

```diff
@@ -1323,6 +1325,8 @@
    <sentence name="PxWebAdminSettingsMainHeaderForTablesTableName" value="Table name"/> 
    <sentence name="PxWebAdminSettingsMainHeaderForTablesStatisticArea" value="Statistics area"/>
    <sentence name="PxWebAdminSettingsMainHeaderForTablesInfo" value="Type of heading that will be used for table pages"/>
+   <sentence name="PxWebAdminSettingsShowExternalSearchLink" value="Show external search"/>
+   <sentence name="PxWebAdminSettingsShowExternalSearchLinkInfo" value="If link to external search page shall be displayed or not"/>
```

Comment:
Should we add something like: Typically the big search on your site ?

Reply:
That is a good idea